packages/web/src/pages/sign-up-page/components/EmailField.tsx

```diff
@@ -1,15 +1,23 @@
-import { ChangeEvent } from 'react'
+import { ChangeEvent, useCallback } from 'react'
 
-import { createEmailPageMessages } from '@audius/common/messages'
+import {
+  confirmEmailMessages,
+  createEmailPageMessages
+} from '@audius/common/messages'
 import { emailSchemaMessages } from '@audius/common/schemas'
-import { route } from '@audius/common/utils'
+import { SIGN_IN_CONFIRM_EMAIL_PAGE } from '@audius/common/src/utils/route'
```

Comment:
import {routes} from '@audius/common/utils' i think and then extract the const from that. def an awkward pattern. basically dont want to import from src.